contracts/StablecoinConverter.sol

```diff
@@ -317,26 +306,16 @@ contract StablecoinConverter is EpochTokenLocker {
                 address owner = previousSolution.trades[i].owner;
                 uint orderId = previousSolution.trades[i].orderId;
                 Order memory order = orders[owner][orderId];
-                uint128 buyVolume = previousSolution.trades[i].volume;
-                uint128 sellVolume = getExecutedSellAmount(
-                    buyVolume,
-                    currentPrices[order.buyToken],
-                    currentPrices[order.sellToken]
-                );
-                addBalance(owner, tokenIdToAddressMap(order.sellToken), sellVolume);
+                (, uint128 sellAmount) = getTradedAmounts(previousSolution.trades[i].volume, order);
```

Comment:
One could create a `TradedVolume` struct with two fields and pass this around instead of having these tuples everywhere. 
Not sure if this would be an improvement.

Reply:
I decided against it, as then we have the very long namings.
`
tradeVolume.buyAmount 
tradeVolume.sellAmount 
`

But yeah, both solutions are possible